apps/protocol-frontend/src/hooks/useContributionMetadataStore.ts

```diff
@@ -0,0 +1,27 @@
+import { useMutation } from '@tanstack/react-query';
+import { INFURA_PROJECT_ID, INFURA_PROJECT_SECRET } from '../utils/constants';
+import { MintedContributionSchemaV1, IPFS } from '@govrn/protocol-client';
+
+const useContributionMetadataStore = () => {
+  const { mutateAsync, isLoading, isError, isSuccess } = useMutation(
+    ['ContributionMetadataStore'],
```

Comment:
I don't think we need to pass a key into the mutations -- what I've seen is that it's more important for the queries due to the invalidation by key. But this'll help with any debugging we want to do with the React Query dev tools. 

If we find that it's helpful we can add keys to our other mutations too